ts/__tests__/IterableInboxMessageListItem.test.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { render } from '@testing-library/react-native'
```

Comment:
For some reason, I'm getting an error here when I run the tests locally

Reply:
I will address the unit tests on a PR basis. This current unit test is not working.